core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -95,15 +95,14 @@
     </div>
   </md-card>
 
-{% if SHOW_TRAINABLE_UNRESOLVED_ANSWERS %}
-  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="isCurrentInteractionTrainable()">
+  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="SHOW_TRAINABLE_UNRESOLVED_ANSWERS && isCurrentInteractionTrainable()">
     <div ng-if="EditabilityService.isEditableOutsideTutorialMode() && !isCurrentInteractionLinear()">
       <button type="button" class="btn btn-default btn-lg oppia-add-response-button protractor-test-open-teach-modal" ng-click="openTeachOppiaModal()">
         + Teach Oppia
       </button>
     </div>
+    
   </md-card>
```

Comment:
@EvsChen, Sorry for miss understanding, I asked you to drop a newline between  `</md-card>` and `<md-card>` i.e, between line 105 & 106.

Reply:
Thanks Sandeep. It was my mistake. Finished now.